showcase/src/app/in-memory-data/services/in-memory-data.service.ts

```diff
@@ -0,0 +1,121 @@
+import {
+	InMemoryDbService,
+	ParsedRequestUrl,
+	RequestInfo,
+	RequestInfoUtilities,
+	ResponseOptions
+} from "angular-in-memory-web-api/interfaces";
+import { Observable } from "rxjs";
+import { Inject, Injectable } from "@angular/core";
+import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
+
+const _cloneDeep: Function = require("lodash/cloneDeep");
+const mockData: any = require("../../../../config/json-server/data.json");
+
+@Injectable()
+export class InMemoryDataService implements InMemoryDbService {
+	public constructor(@Inject(STARK_LOGGING_SERVICE) public logger: StarkLoggingService) {
+		/* empty constructor */
+	}
+
+	/**
+	 * Method called by the "angular-in-memory-web-api" to create the "database" hash whose keys are collection names and
+	 * whose values are arrays of collection objects to return or update
+	 * @param _reqInfo - The RequestInfo object in case this method is invoked as a result of a POST `commands/resetDb` request
+	 * @returns The "database" object or an Observable of Promise that will return such object asynchronously
+	 * @link https://github.com/angular/in-memory-web-api#basic-setup
+	 */
+	public createDb(_reqInfo?: RequestInfo): {} | Observable<{}> | Promise<{}> {
+		// replace the "uuid" field defined in the mock data by the "id" field expected by the in-memory-db
+		this.deepReplaceProperty(mockData, "uuid", "id");
+
+		// 	IMPORTANT: cannot mock "logging" and "logout" requests since they are performed via XHR and not via Angular
+		return mockData;
+	}
+
+	/**
+	 * Custom implementation of the "parseRequestUrl" method which will be called by the "angular-in-memory-web-api"
+	 * @param _url - The request URL
+	 * @param _requestInfoUtils - Some utility methods provided by the "angular-in-memory-web-api" library including the default parser.
+	 * @returns The ParsedRequestUrl object in case the original one was modified or "undefined" to let the service use the default parser.
+	 * @link https://github.com/angular/in-memory-web-api#custom-parserequesturl
+	 */
+	public parseRequestUrl(_url: string, _requestInfoUtils: RequestInfoUtilities): ParsedRequestUrl | undefined {
+		/// this.logger.debug("PARSE_REQUEST_URL -> ", _requestInfoUtils.parseRequestUrl(_url));
```

Comment:
Is this comment relevant ?

Reply:
Yes, just to know that we can use that method, but I'll change the comment so it is clear why is there